cosmos/utils.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
```

Comment:
Let's avoid introducing generic modules at the root of Cosmos. I've seen `utils.py` becoming huge files with all sorts of things in previous projects.
What do you think if we add this to `dbt/project.py` instead? I'm open to other suggestions!

Reply:
This has been updated, agree that a generic `utils` can be bloated - thanks for the suggestion.